sdks/builds/ios.mk

```diff
@@ -370,7 +370,13 @@ _ios-$(1)_CONFIGURE_FLAGS= \
 .stamp-ios-$(1)-toolchain:
 	touch $$@
 
-.stamp-ios-$(1)-$$(CONFIGURATION)-configure: | $(if $(IGNORE_PACKAGE_LLVM),package-llvm-$(3),download-llvm-$(3))
+.PHONY: clean-ios-llvm-$(3)
+clean-ios-llvm-$(3): clean-llvm-$(3)
+
+.PHONY: build-ios-llvm-$(3)
+build-ios-llvm-$(3): $(if $(IGNORE_PACKAGE_LLVM),download-llvm-$(3),package-llvm-$(3))
```

Comment:
It's my understanding that any target of the form `build-ios-llvm` would conflict with the `build-%` target. 

I was asked to restore the build-ios-llvm and clean-ios-llvm targets. Should I instead inform XI of API changes for @BrzVlad 's integration?

Reply:
@alexanderkyte where were you asked? what's the use case? I don't have all the context you have, please make sure I have it as well if you want me to understand better.

Reply:
When @BrzVlad was doing the iOS integration, he noticed that we removed the build-ios-llvm and clean-ios-llvm targets in the sdks that the XI build depends upon. He asked me to restore the behavior of the targets. 

After identifying that these targets had the effect of building llvm for all enabled targets, I made each individual `build-ios-llvm-$(3)` be triggered by the top-level target.

Your comment makes it seem that you feel that the existence of this target at all is a design problem because the caller no longer specifies the platform. 

So it seems that either @BrzVlad needs to modify the code using this target, or we need to restore the target. 

@BrzVlad, do you think it would be very difficult to make that change?

Reply:
You should have the `build-ios-llvm-{llvm32,llvm64}` in XI directly, defined as follow:
```
build-ios-llvm-%:
    $(MAKE) -C external/mono/sdks/builds download-ios-llvm-$* || $(MAKE) -C external/mono/sdks/builds package-ios-llvm$*
```

Reply:
Are all stakeholders okay with me closing this PR then?